library/src/main/java/com/google/android/material/motion/MotionRuntime.java

```diff
@@ -60,4 +63,9 @@ public void next(T value) {
     Interaction<O, T> interaction, O target, ConstraintApplicator<T> constraints) {
     interaction.apply(this, target, constraints);
   }
+
+  /* Returns an ReactiveView */
```

Comment:
Can you copy the documentation from swift?
> Returns a reactive version of the given object and caches the returned result for future access.

Also, javadoc should start with `/**`